platform/mbed_lib.json

```diff
@@ -34,6 +34,11 @@
         "force-non-copyable-error": {
             "help": "Force compile time error when a NonCopyable object is copied",
             "value": false
+        },
+
+        "poll-use-lowpower-timer": {
```

Comment:
events have something similar:

```
        "use-lowpower-timer-ticker": {
            "help": "Enable use of low power timer and ticker classes. May reduce the accuracy of the event queue.",
            "value": 0
        }
```

these 2 are similar in functionality, should be aligned

Reply:
I tried to align it as much as possible

Why not `use-lowpower-timer`? This would imply that the whole `platform` uses `LowPowerTimer`

Why not `poll-use-lowpower-ticker-timer`? There is no ticker here, unlike in `events`